sdk/identity/azure-identity/TROUBLESHOOTING.md

```diff
@@ -244,6 +248,14 @@ Get-AzAccessToken -ResourceUrl "https://management.core.windows.net"
 
 >Note that output of this command will contain a valid access token, and SHOULD NOT BE SHARED to avoid compromising account security.
 
+## Troubleshoot Multi Tenant Authentication Issues
```

Comment:
```suggestion
## Troubleshoot multi-tenant authentication issues
```

Reply:
Since the preference seems to be for sentence-case as opposed to title-case, should I update the other headings for consistency in the document?

Reply:
Yes. The Microsoft writing style guide calls for sentence casing.